Bottom: aeb9305e3535eefa172def2f985caca4ddfb058d
Top:    949f9d0eb39d8568be2b8bb051296d7277999143
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-30 10:41:51 +0000

Refresh of fix-auto-enforce-drc-with-spli

---

diff --git a/src/action.c b/src/action.c
index 116bb83..1d6d8f9 100644
--- a/src/action.c
+++ b/src/action.c
@@ -1314,7 +1314,8 @@ NotifyMode (void)
 		  PCB->Clipping ^= 3;
 		}
 	    }
-          LookupConnection (Note.X, Note.Y, true, 1, CONNECTEDFLAG, false);
+	  if (TEST_FLAG (AUTODRCFLAG, PCB) && !TEST_SILK_LAYER (CURRENT))
+	    LookupConnection (Note.X, Note.Y, true, 1, CONNECTEDFLAG, false);
 	  Draw ();
 	}
       break;
